flocker/node/agents/test/test_blockdevice.py

```diff
@@ -169,7 +173,12 @@
 
 DESIRED_DATASET_ATTRIBUTE_STRATEGIES = {
     'dataset_id': uuids(),
-    'maximum_size': integers(min_value=0).map(
+    'maximum_size': integers(
+        min_value=0,
+        max_value=(
+            sys.maxint - LOOPBACK_MINIMUM_ALLOCATABLE_SIZE
+        ) // LOOPBACK_ALLOCATION_UNIT,
```

Comment:
The annoying thing about this is we have tests that actually try to create loopback devices of size maxint.... but that's not a regression that you added. I think what you added is compatible with what was originally here. Although we should possibly re-thing this whole thing.